### PR TITLE
docs(scully): translate docs\Reference\plugins\community-plugins\cano…

### PR DESCRIPTION
**Pull request by AlexScigalszky**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/scullyio/scully/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Other... Please describe: add spanish translations

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: docs/Reference/plugins/community-plugins/canonical_es.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+# Complemento `canonical`
+
+<div class="docs-link_table">
+  <a class="homepage" href="https://github.com/pjlamb12/scully-plugin-canonical"></a>
+  <a class="repository" href="https://github.com/pjlamb12/scully-plugin-canonical"></a>
+</div>
+
+## Visión general
+
+El proposito del complemento canónico de de Scully es tomar un atributo link canónico de la ruta y replazar el valor de ese enlace canónico en el encabezado de la página, o agregar la etiqueta link si no existe. Esto es necesario para propósits de SEO, si un artículo se publicó primero en otro sitio, por ejemplo.
```
Comment (Villanuevand) [inline docs/Reference/plugins/community-plugins/canonical_es.md:17]:
```suggestion
El propósito del complemento canónico de de Scully es tomar un atributo link canónico de la ruta y reemplazar el valor de ese enlace canónico en el encabezado de la página, o agregar la etiqueta link, si no existe. Esto es necesario para propósitos de SEO, si un artículo se publicó primero en otro sitio, por ejemplo.
```

File: docs/Reference/plugins/community-plugins/canonical_es.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+## Caraterísticas
+
+- ✅ Reemplaza el link canónico con una url con la url provista para el tag link
```
Comment (Villanuevand) [inline docs/Reference/plugins/community-plugins/canonical_es.md:21]: Could you try to add sense to this line?. Is hard to understand.

File: docs/Reference/plugins/community-plugins/canonical_es.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+`npm install scully-plugin-canonical --save-dev`
+
+### Yarn
+
+`yarn add scully-plugin-canonical --dev`
+
+## Uso
+
+Los complementos Render obtiene información de la ruta, y busca el título en la siguientes ubicaciones, en orden:
```
Comment (Villanuevand) [inline docs/Reference/plugins/community-plugins/canonical_es.md:42]:
```suggestion
Los complementos Render obtienen información de la rutas, y buscan el título en la siguientes ubicaciones, en orden:
```

**Review by Villanuevand — CHANGES_REQUESTED** [review]
Just minor changes, 

Thanks to contribute with  Scully!

File: docs/Reference/plugins/community-plugins/canonical_es.md (hunk 2 above)
Comment (AlexScigalszky) [reply docs/Reference/plugins/community-plugins/canonical_es.md]: done

**Review by Villanuevand — APPROVED** [review]
Looks good! 

Thanks to contribute with Scully!

**Pull request merged**